test/index.js

```diff
@@ -188,6 +188,17 @@ function isHidden(filePath) {
                 // If sanitation were done, 'class' property should be removed.
                 assert.equal(React.renderToStaticMarkup(vdom), '<div><pre><code class="language-empty"></code></pre></div>');
             });
+
+            it('passes toHast options to inner toHAST() function', function() {
+                var markdown = '# Foo';
+
+                var vdom = remark().use(reactRenderer, {
+                    createElement: React.createElement,
+                    toHast: {allowDangerousHTML: true}
+                }).process(markdown, {}).contents;
```

Comment:
That is true - sanitize: false disables [hast-util-sanitize](https://github.com/wooorm/hast-util-sanitize), which has a different set of rules than `allowDangerousHtml` - the latter allows elements verbatim and the former fixes potentially-dangerous properties that are legal in Markdown.